README.md

```diff
@@ -3,20 +3,68 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-{Project tag line}
+Chinook.Navigation
 
-{Small description of the purpose of the project}
+This library offers a simple method of navigating through an app. [StackNavigation](src/StackNavigation.Abstractions/StackNavigation.md) and [SectionsNavigation](src/SectionsNavigation.Abstractions/SectionsNavigation.md) are used together to province a simple interface to navigate forward, backward and between discrete sections.
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 
 ## Getting Started
 
-{Instructions to quickly get started using the project: pre-requisites, packages
-to install, sample code, etc.}
+### Uno project
+
+To get started in a project which uses the [Uno platform](https://platform.uno/):
+
+* Add to your project the `Chinook.SectionsNavigation.Uno` nuget package and its dependencies.
+* Create a singleton instance of `IStackNavigator` which will be used throughout the app:
+
+```
+var shell = Shell.Instance;
+
+// Implement GetRegistrations() so that it returns Dictionary<business object, app page>
+Dictionary<Type, Type> registrations = GetRegistrations();
+
+// This is instance which should be used throughout the app
+ var navigator = new FrameSectionsNavigator(shell.NavigationFrame, registrations);
+```
+
+### Unit tests
+
+To support navigation in your unit tests:
+
+* Add to your project the `Chinook.SectionsNavigation` nuget package and its dependencies.
+* For each test which involves navigation, create an instance of BlindSectionsNavigator:
+
+```
+var navController = new BlindSectionsNavigator("section name");
+```
+
+For additional information on how to use these navigators, please take a look at these documents:
+
+[StackNavigation](src/StackNavigation.Abstractions/StackNavigation.md)
+
+[SectionsNavigation](src/SectionsNavigation.Abstractions/SectionsNavigation.md)
 
 ## Features
 
-{More details/listing of features of the project}
+### No double navigation
+If you invoke 2 operations simultaneously (double tap, press 2 buttons with 2 fingers, etc.), only the first will actually run.
+This is because the the request state (either processing or processed) is part of the `ISectionsNavigator.State`.
+If a request is made while another is processing, the second request is cancelled.
+
+### Background navigation
+You can navigate in sections that are not active. This is useful if you want to _prepare_ a section before entering it.
+
+### Stack navigator adapter
+You can use `SectionsNavigatorToStackNavigatorAdapter` to adapt a `ISectionsNavigator` into a `IStackNavigator`. This is useful if your app mostly use the single stack pattern.
+
+### Customize animations
+When using `FrameSectionsNavigator`, you can customize or remove the animations of the MultiFrame using `MultiFrame.Animations`.
+
+### Navigation testing
```

Comment:
Maybe add a section about modals? I think it's first class citizens

Reply:
done!